lib/smart_answer/flow.rb

```diff
@@ -16,18 +16,18 @@ def initialize(&block)
       instance_eval(&block) if block_given?
     end
 
+    def append(flow)
+      flow.nodes.each do |node|
+        node.flow = self
```

Comment:
I'm not really clear why you think this might not play nicely with the `FlowRegistry`.

The appending of the nodes from another "inner" flow is just an implementation detail of the normal `Flow.build` method. I think the "inner" flow instance would just exist temporarily during the building of the "outer" flow after which it would be available for garbage collection. The "outer" flow would be cached in the `FlowRegistry` as usual.

Can you be any more specific about your concern?